ci/jobs/pull_request.yml.j2

```diff
@@ -0,0 +1,15 @@
+jobs:
+- name: dataworks-metrics-infrastructure-pr
+  plan:
+  - get: dataworks-metrics-infrastructure-pr
+    trigger: true
+  - .: (( inject meta.plan.terraform-bootstrap ))
+    config:
+      inputs:
+        - name: dataworks-metrics-infrastructure-pr
+  - .: (( inject meta.plan.terraform-plan ))
+    config:
+      params:
+        AWS_ACC: "{{accounts['management-dev']}}"
```

Comment:
this should not be done like this, please source these values from Concourse's secrets engine

Reply:
Happy to fix this, but this PR was marked WIP, so potentially everything in it is subject to change. Once it is no longer marked WIP it will make sense to review and I would then appreciate feedback.